src/app/checkout/step-2/bank-account/bank-account.tpl.html

```diff
@@ -82,7 +82,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Bank Account Pay
             <div class="panel-body">
               <div class="row">
                 <div class="col-sm-8">
-                  <img src="/assets/img/check-sample.png" alt="Credit Card Icon"/>
+                  <img src="/assets/images/check-sample.png" alt="Credit Card Icon"/>
```

Comment:
You can add another variable to angular environment if you want. You'd have to inject it and put it on the template scope which is annoying but I think you'd have to do that with any global angular service/constant unless you put it on window.